vipps-partner-api.md

```diff
@@ -266,6 +268,19 @@ Here is a sample request:
 }
 ```
 
+**Improtant:** Please provide all the required fields, so it will not be necessary for
+merchants to request more details. This is the most
+[typical reason for delays](https://developer.vippsmobilepay.com/docs/vipps-partner/#typical-reasons-for-delays).
+
+We have made as many of the fields as possible optional, but please
+try to send as much as you can, to make it easy for the merchant.
+
+**Please note:** The merchant can not change the information provided by the partner, so if
+something needs to be corrected, the merchant must contact the partner to have
+the partner submit a new pre-fill product order with the correct details.
```

Comment:
This is not correct, if I remember correct, just some of the fields are locked, such a what partner and what price.

Reply:
When was it changed? Can you check which fields, and list them?

Reply:
Just checked, everything is editable except what partner(always) and what pricePackage(if provided by partner)
<img width="1264" alt="Screenshot_20230609_141359" src="https://github.com/vippsas/vipps-partner-api/assets/7635783/db2b2a7b-62ba-4e23-81e3-7edecc88bd2d">

Reply:
This has always been the case.

Reply:
I updated it now. Does it look ok?